src/controllers/fridges/fridge.controller.ts

```diff
@@ -0,0 +1,48 @@
+import { createFridge } from './handlers/create.fridge.handler';
+import { Authorized, Delete, Get, JsonController, Param, Patch, Post, Req } from 'routing-controllers';
+import { ArrayRepresenter, Body, ListRepresenter, Query, Representer, StatusCode } from '@panenco/papi';
+import { OpenAPI } from 'routing-controllers-openapi';
+import { FridgeBody, FridgeGetBody, FridgeGiftBody } from '../../contracts/fridge.body';
+import { CreateFridgeView } from '../../contracts/fridge.view';
+import { ProductView } from '../../contracts/product.view';
+import { getFridge } from './handlers/get.fridge.handler';
+import { SearchQuery } from '../../contracts/search.query';
+import { giftFridge } from './handlers/gift.fridge.handler';
+import { deleteFridge } from './handlers/delete.fridge.handler';
+import { Request } from 'express';
+import { adminRequirement } from '../../contracts/admin.requirements/admin.requirement';
+
+@JsonController("/fridges")
+
+export class FridgeController {
```

Comment:
What about the update endpoint?

Reply:
Also, it would be valuable to have an endpoint to list all the fridges available